docs/DeveloperGuide.md

```diff
@@ -304,10 +304,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. DoConnek Pro shows an error message.
 
       Use case resumes at step 2.
 
+**Use case: Add a patient**
+
+**MSS**
+
+1. User searches list to check if patient is already in DoConnek Pro
+2. DoConnek Pro shows that there are no entries matching the patient's name
```

Comment:
Since we already have a statement stating that **System** is the `DoConnek Pro` above,
perhaps we can consider using "System" instead of DoConnek Pro here instead?

Reply:
Makes sense, I have changed it!